examples/cluster/cluster.yaml

```diff
@@ -19,3 +23,23 @@ metadata:
 spec:
   region: ${AWS_REGION}
   sshKeyName: ${SSH_KEY_NAME}
+  networkSpec:
+    subnets:
+    - availabilityZone: ${AWS_REGION}a
+      cidrBlock: "10.0.100.0/24"
```

Comment:
Given that we use `10.0.0.0/16` by default, can we use the first subnets in the range http://www.davidc.net/sites/default/subnets/subnets.html?network=10.0.0.0&mask=16&division=23.ff4600 ?

Reply:
We really shouldn't overlap this with the pod network.

Reply:
> See above, I think the changes to the cluster configuration aren't necessary

yep, in fact, I hope that the generate script can generate the ha cluster yaml with amazon-vpc-cni or with  calico